### PR TITLE
Admission Controllers 101 Docs

### LINKED ISSUE #857: [Enhancement] Admission Controllers 101
### Description

It would be nice to have a section in the docs which is a better, easier, more comprehensive introduction to admission controllers with some nice, perhaps animated, graphics so users get more comfortable with what they're installing and how generally they work.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by 1Shubham7**
## Related issue #

Admission Controllers is a part of Introduction section of the Docs. Closes #857 

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by 1Shubham7** [comment]
Created this new PR and closed previous one since commits were not signed there. @anushkamittal2001 and @chipzoller please give it a review

**Comment by chipzoller** [comment]
I will review this again today.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: content/en/docs/Introduction/admission-controllers.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- **Governance:** Using Admission controllers you can enforce the adherence to certain practices like having good labels, annotations, resource limits, or other settings. For example, you can enforce label validation on different objects to ensure proper labels are being used, such as every object being assigned to a team or project, or every deployment specifying an app label. In another example, You can enforce annotations automatically being added to objects, such as attributing the correct cost center for a “dev” deployment resource.
+
+- **Configuration management:** Admission controllers allow you to validate the configuration of the objects running in the cluster and prevent any obvious misconfigurations from hitting your cluster. Admission controllers can be useful in detecting and fixing images deployed without semantic tags, such as by automatically adding resource limits or validating resource limits, ensuring reasonable labels are added to pods, or ensuring image references used in production deployments are not using the latest tags, or tags with a -dev suffix.
```
[the remark this answers is not in the document]
Comment (1Shubham7) [reply content/en/docs/Introduction/admission-controllers.md]: I have covered use cases outlined in your presentation and am assuming that you know some of these use cases are more specific to Kyverno while the broader topic we are addressing is admission controllers. Also please comment if we should remove the previously covered usecases.

**Comment by 1Shubham7** [comment]
Hey @chipzoller, I have added use cases you outlined in your presentation. Also use cases like "Fined-grained RBAC" and "ops automation", I found them to be more specific to Kyverno than admission controllers in general so haven't added them. Please give it a review.

**Comment by 1Shubham7** [comment]
@chipzoller, please have a look.

**Comment by chipzoller** [comment]
> Hey @chipzoller, I have added use cases you outlined in your presentation. Also use cases like "Fined-grained RBAC" and "ops automation", I found them to be more specific to Kyverno than admission controllers in general so haven't added them. Please give it a review.

"ops automation", yes, is more Kyverno specific, but fine-grained RBAC is not. All validating admission controllers for the most part have this capability and is a major use case.

**Comment by 1Shubham7** [comment]
> "ops automation", yes, is more Kyverno specific, but fine-grained RBAC is not. All validating admission controllers for the most part have this capability and is a major use case.

yes, I agree "fine grained RBAC" is a major use, now that I studied about it. I misunderstood it to be something else. added it as well.

**Review by chipzoller — APPROVED** [review]
Thanks for kicking this off. I'm going to build on your content here.

**Pull request merged**